components/tokens/texts/h1.vue

```diff
@@ -0,0 +1,9 @@
+<template>
+  <h1><slot /></h1>
```

Comment:
Out of curiosity, what's the reasoning behind all those HTML tags as vue components (`<h*>`, `<div>`, `<p>`)?

Reply:
it's for scope them as in components base architecture you want them to be isolated. I dont want to have too much global css